pkg/util/mount/nsenter_mount.go

```diff
@@ -203,6 +204,33 @@ func (n *NsenterMounter) IsLikelyNotMountPoint(file string) (bool, error) {
 	return true, nil
 }
 
+// Calling Open with O_EXCL without O_CREATE only supported on linux, so
+// we just return false here.
+func (n *NsenterMounter) DeviceOpened(pathname string) (bool, error) {
+	if isDevice, err := n.PathIsDevice(pathname); !isDevice {
+		return false, fmt.Errorf(
+			"PathIsDevice failed for path %q: %v",
+			pathname,
+			err)
+	}
+	return false, nil
+}
+
+// Uses FileInfo returned from os.Stat to check if path refers to a device
+func (n *NsenterMounter) PathIsDevice(pathname string) (bool, error) {
```

Comment:
Instead of duplicating the code, put the code in a helper method in mount_linux.go and have both places call it. Like `List(...)` does.